homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -82,6 +82,7 @@ class PlenticoreSensorEntityDescription(
         name="Home Power from Battery",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
For reviewer: This was the main reason to change: This is a measurement entity which measures power to the home from the battery.